dangerfile.ts

```diff
@@ -11,7 +11,11 @@ message('Removed files in this PR: \n - ' + removedFiles)
 
 // Commits should start with feat, fix, major or chore
 danger.git.commits.forEach(commit => {
-  if (!commit.message.match(/^(feat:)|(fix:)|(major:)|(chore:)|(docs:)/g)) {
+  if (
+    !commit.message.match(
+      /^(feat:)|(fix:)|(major:)|(chore:)|(docs:)|Merge branch 'master'/g
```

Comment:
I don't really like this but if it stops this being barrier to getting this rolled out then that's good